test/test_browser.py

```diff
@@ -2612,6 +2612,15 @@ def test_html5_webgl_create_context(self, args):
   def test_html5_webgl_create_context2(self):
     self.btest_exit('webgl_create_context2.cpp')
 
+  @requires_graphics_hardware
+  # Verify bug https://github.com/emscripten-core/emscripten/issues/22943: creating a WebGL context with explicit swap control and offscreenCanvas
+  @parameterized({
+    'offscreencanvas': (['-sOFFSCREENCANVAS_SUPPORT'],),
+    'offscreenframebuffer': (['-sOFFSCREEN_FRAMEBUFFER', '-DUSE_OFFSCREEN_FRAMEBUFFER'],),
+    })
```

Comment:
Flake8 doesn't like this `})`.  It should be indented to match the @parameterized above.  

Sorry for all the back a forth, I think this will be good to go after this fix.

Reply:
(I fixed this now)